bonsai/Data/Entity/Budget.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Budget.swift
+//  bonsai
+//
+//  Created by hoang on 04.01.2022.
+//
+
+import Foundation
+import CoreData
+
+@objc(Budget)
+public class Budget: NSManagedObject, Identifiable, BudgetProtocol {
+   @NSManaged public var id: UUID
+   @NSManaged public var name: String
+   @NSManaged public var amount: Double // NSDecimal
+   @NSManaged public var period: Int64
```

Comment:
I'm thinking about adding a little bit of explicity here by renaming to `periodDays`, so we always read it correctly